Bio/PDB/MMCIFParser.py

```diff
@@ -57,6 +57,7 @@ def get_structure(self, structure_id, filename):
          - filename - name of mmCIF file, OR an open text mode file handle
 
         """
+
```

Comment:
You need to remove this new blank line, it is one of the reason the style checks on TravisCI are failing:

```
$ flake8 Bio/
Bio/PDB/MMCIFParser.py:52:1: D202 No blank lines allowed after function docstring
...
```